index.html

```diff
@@ -698,12 +704,48 @@ <h3>
             </p>
           </li>
         </ol>
+        <p data-dfn-for="PerformanceResourceTiming">
+          The <dfn>renderBlockingStatus</dfn> getter steps are to return
+          <a data-link-for="RenderBlockingStatusType">blocking</a>
+          if <a>this</a>'s <a data-for="PerformanceResourceTiming">timing
+          info</a>'s [=fetch timing info/render-blocking=] is true
+          else <a data-link-for="RenderBlockingStatusType">non-blocking</a>
+          and the <a>relevant global object</a> for <a>this</a>.
```

Comment:
I don't understand this line, how is the global object used here?

Reply:
It's not. Apologies for the confusion. Removed.